src/app/components/quotes/quotes.hbs

```diff
@@ -1,36 +1,36 @@
 <style scoped>
-    @import url(/app/pages/home/quotes/quotes.css);
+    @import url(/app/components/quotes/quotes.css);
 </style>
 <div class="quotes">
     <div class="quote-bucket right">
-        <img src="/images/home/quotes/juan.jpg" width="175" height="230" />
+        <div class="image juan"></div>
```

Comment:
This is clearly placeholder text/images. Do you want "juan" class, or something more structural like first-quote? Or maybe use pseudo-selectors like :nth-child on the quote-bucket div?